keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -272,6 +273,28 @@ Thus, this KEP largely the plan described [here](#plan), with some changes:
 
 #### Open Questions
 1. For the newly introduced CRI API fields, should there be Windows and Linux specific fields?
+    For the alpha implementation, [PR #102789](https://github.com/kubernetes/kubernetes/pull/102789) added a Linux specific field `LinuxPodSandboxStats`. It also added Windows Specific field called  `WindowsPodSandboxStats` but left it blank.  
+
+    The proposal for Widnows support is to add a fields to `PodSandboxStats` which will be used by both Linux and Windows.  Windows and
+    Linux would both use this field filling in stats that make most sense for them.  Windows is currently missing several stats but can reasonably fill in many of the missing fields (see the [Windows](#windows) section below).  This would be similar approach that is use today for the current CRI `ListContainerStats` call and this would make logic in kubelet more straight forward.
+
+    The field `LinuxPodSandboxStats` would stay the same (with duplicated fields) for backwards compatibility.  We will also leave `WindowsPodSandboxStats` with a comment that it is not currently used.   This option allows us to add specific fields in the future, Linux and Windows sub fields could be added to `LinuxPodSandboxStats` and `WindowsPodSandboxStats` if there are use cases for specific fields.  
```

Comment:
the fields being marked as deprecated conflicts with this. I am open to the direction. To me, if we're keeping linux specific fields in the top level, then it makes sense to me to include windows specific fields in the top level too.

Reply:
I missed updating this section based on https://github.com/kubernetes/enhancements/pull/3439#pullrequestreview-1056406603 comment. 

I was originally thinking we could keep the `LinuxPodSandboxStats` and `WindowsPodSandboxStats` which allow for OS specific fields outside the generic ones we've defined in the KEP here.  We don't have specific use cases for this right now so not sure if it makes sense.  

I think we have a couple choices: 

- designate `LinuxPodSandboxStats` and `WindowsPodSandboxStats` as deprecated 
- designate the individual fields in  `LinuxPodSandboxStats` deprecated and keep `LinuxPodSandboxStats` and `WindowsPodSandboxStats` for future OS specific use cases.

Reply:
If we're leaving rss_bytes in the top-level, despite techincally being linux specific, then I think we should deprecate and drop the linux and windows specific fields.

Reply:
(deprecate and drop*)

Reply:
no longer the direction we are taking.